src/replication/status.cpp

```diff
@@ -56,7 +76,14 @@ void from_json(const nlohmann::json &j, ReplicationRoleEntry &p) {
       auto json_epoch = j.value(kEpoch, std::string{});
       auto epoch = ReplicationEpoch{};
       if (!json_epoch.empty()) epoch.SetEpoch(json_epoch);
-      p = ReplicationRoleEntry{.version = version, .role = MainRole{.epoch = std::move(epoch)}};
+      std::optional<utils::UUID> main_uuid;
+      try {
+        main_uuid.emplace(j.at(kMainUUID));
+      } catch (std::exception &e) {
```

Comment:
This can also fail because of emplace, so I think only fetching kMainUUID should be in try-catch block